core/src/test/scala/org/apache/spark/sql/expression/index/UnsignedTestSuite.scala

```diff
@@ -112,7 +112,7 @@ class UnsignedTestSuite extends BaseTiSparkTest {
       List(List(1), List(2), List(3)),
       List(List(0), List(1), List(2), List(3), List(unsignedLongMaxValue))
     )
-    assert(queries.size == answers.size)
+//    assert(queries.size == answers.size)
```

Comment:
why

Reply:
It is commented for speeding up debug process.

Reply:
if this is commented, the following lines may break.

Reply:
if this is commented, the following lines may break.